.gitignore

```diff
@@ -2,12 +2,16 @@
 /.gradle/
 /build/
 src/main/resources/docs/
+/bin/
```

Comment:
Just as clarification, is this addition due to VSCode?

Reply:
No, the `/bin/` folder is where Gradle puts the Java class files after compiling. I don't believe these class files should be version controlled; `/bin/` was included in the `.gitignore` file in the iP, so I'm not sure why it wasn't included for the tP

Reply:
Ah... is that the same for the /.vscode/ lines on line 12-14?

```
# VS Code files
/.vscode/

```

Reply:
Umm not really, the `/.vscode/` line is to ignore VS Code's config files (I use VS Code instead of IntelliJ). It's not related to the Java class files.

Reply:
Ahh okay thanks for the clarification!